File: packages/sdk-py/src/grantex/manifests/langsmith.py
from grantex.manifest import ToolManifest, Permission

manifest = ToolManifest(
    connector="langsmith",
    description="LangSmith Observability API",
    tools={
        "list_runs": Permission.READ,
        "get_run": Permission.READ,
        "get_run_stats": Permission.READ,
        "list_datasets": Permission.READ,
        "create_feedback": Permission.WRITE,
    },
)

File: packages/sdk-ts/src/manifests/langsmith.ts
import { ToolManifest, Permission } from '../manifest.js';

export const langsmithManifest = new ToolManifest({
  connector: 'langsmith',
  description: 'LangSmith Observability API',
  tools: {
    list_runs: Permission.READ,
    get_run: Permission.READ,
    get_run_stats: Permission.READ,
    list_datasets: Permission.READ,
    create_feedback: Permission.WRITE,
  },
});
